### PR TITLE
UIIN-1682 create Jest/RTL test for ItemsListContainer.js

### PR DESCRIPTION
**Pull request by cchitneedi**
Refs:
UIIN-1682
create Jest/RTL test for ItemsListContainer.js

URL: https://issues.folio.org/browse/UIIN-1682

### CODE REVIEW

**Review by ssandupatla — APPROVED** [review]

File: src/Instance/ItemsList/ItemsListContainer.test.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  });
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+  it('isFetching should be true', async () => {
+    await act(async () => {
+      const { container } = renderWithIntl(<ItemsListContainerSetup />, translations);
+      const spinner = container.querySelector('.spinner');
+      expect(spinner).toBeInTheDocument();
+      expect(spinner).toHaveStyle('width: 60px; height: 30px;');
```
Comment (ryandberger) [inline src/Instance/ItemsList/ItemsListContainer.test.js:85]: This test is too specific making it fragile if we make any adjustments to the spinner's CSS. Please remove it.

**Review by ryandberger — CHANGES_REQUESTED** [review]
Please address my previous comment.

File: src/Instance/ItemsList/ItemsListContainer.test.js (hunk 1 above)
Comment (cchitneedi) [reply src/Instance/ItemsList/ItemsListContainer.test.js]: yes, removed the adjustments and Thank you @ryandberger

**Review by zburke — CHANGES_REQUESTED** [review]

File: src/Instance/ItemsList/ItemsListContainer.test.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+      const bounce1 = container.querySelector('.bounce1');
+      const bounce2 = container.querySelector('.bounce2');
+      const bounce3 = container.querySelector('.bounce3');
+      expect(bounce1).toBeInTheDocument();
+      expect(bounce2).toBeInTheDocument();
+      expect(bounce3).toBeInTheDocument();
+    });
+  });
```
[the remark this answers is not in the document]
Comment (cchitneedi) [reply src/Instance/ItemsList/ItemsListContainer.test.js]: Hi @zburke ,
we have mocked the Loading from @folio/stripes/components and updated the file

Thanks and Regards,
Charan

**Review by ryandberger — APPROVED** [review]

**Review by VSnehalatha — APPROVED** [review]

**Review by ryandberger — APPROVED** [review]

**Pull request merged**